lib/fastlane_core/simulator.rb

```diff
@@ -3,7 +3,7 @@
 module FastlaneCore
   class Simulator
     class << self
-      def all
+      def all(requested_os_type: 'iOS')
```

Comment:
Should we do some kind of validation for this? Or how about, we subclass `Simulator` to be `SimulatorTV` or similar, so that the user doesn't have to know the actual string, but just the class name.